Week1/homework/todo.sql

```diff
@@ -0,0 +1,74 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+-- MY Database 
+
+-- -----------------------------------------------------
+-- Schema mydb
+--
+-- MY Database 
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 COLLATE utf8_bin ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Users`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Users` (
+  `idUsers` INT NOT NULL,
+  `First name` VARCHAR(45) NOT NULL,
+  `last name` VARCHAR(45) NOT NULL,
+  `Birthday` VARCHAR(45) NOT NULL,
+  `E-mail adress` VARCHAR(45) NOT NULL,
+  PRIMARY KEY (`idUsers`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Tags`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Tags` (
+  `idTags` INT NOT NULL,
+  `Homework` VARCHAR(45) NOT NULL,
+  `sport` VARCHAR(45) NOT NULL,
+  `reading` VARCHAR(45) NOT NULL,
```

Comment:
Instead of the three columns `homework` ,`sport` and `reading`, you could have one column called `tag` and you can add homework, sport and reading as records in the table.